airflow/decorators/task_group.py

```diff
@@ -184,6 +184,7 @@ def task_group(
     ui_color: str = "CornflowerBlue",
     ui_fgcolor: str = "#000",
     add_suffix_on_collision: bool = False,
+    max_active_groups_per_dagrun: int | None = None,
```

Comment:
How would this work with nested task group?

Reply:
Note that nested expansion may still happen at some point if someone asks for it. It’d be best if this is designed in a way so it leaves enough room to avoid a rewrite in the future.